rfcs/text/0030/output.yml

```diff
@@ -0,0 +1,48 @@
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# 	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+- name: output
+  title: Output
+  group: 2
+  short: Fields describing a chunk of tty output generated by a process.
+  description: >
```

Comment:
If this fieldset is super specific to a process, do we think we should instead be adding these directly to process as well? 

I hesitate as we don't want to make process super bloated, but that super specificity gives me pause for giving it its own fieldset.

Reply:
@m-sample any reservations about putting output at process.output as a nested field? This would avoid having to make it a top level field. Also I think @ferullo had some thoughts about the naming for the output event, and whether it should be event.action: 'io' or event.action: 'tty_io'.  Future proof for if we wanted to capture stdin/input?

Reply:
process.io would be OK - it will not present in other process though which might be confusing - e.g. proces.session_leader.io etc would never be populated.      IO is the best generalization as this could be read or writes of data to any fd of interest in the long term.  

For now we are focusing on TTY output because that is a generally plaintext/readable (ie not encrypted like most TCP socket fd traffic via TLS),  TTY's mostly echo their input as output (so no need to capture input for now), and TTYs are an exfil vector to be watched.